index.html

```diff
@@ -623,14 +613,12 @@ <h2>
           basis.
         </p>
         <p>
-          This group primarily conducts its technical work <i class="todo">pick
-          one, or both, as appropriate:</i> on the public mailing list
-          <a class="todo" id="public-name" href=
-          "mailto:public-TODO@w3.org">public-<i class=
-          "todo">[email-list]</i>@w3.org</a> (<a class="todo" href=
-          "https://lists.w3.org/Archives/Public/public-TODO/">archive</a>)
-          <i class="todo">or</i> on <a class="todo" id="public-github" href=
-          "TODO">GitHub issues</a>. The public is invited to review, discuss
+          This group primarily conducts its technical work  on the public mailing list
+          <a id="public-name" href=
+          "mailto:public-networks-ig@w3.org">public-networks-ig@w3.org</a> (<a class="todo" href=
+          "https://lists.w3.org/Archives/Public/public-networks-ig/">archive</a>)
+          and on <a id="public-github" href=
+          "https://github.com/w3c/web-networks/issues">GitHub issues</a>. The public is invited to review, discuss
```

Comment:
Should it be .../web-networks or .../web-networks-charter ?

Reply:
I expect the IG will use a separate repository for its discussions than the one used for the charter itself